packages/ckeditor5-ui/theme/globals/_evaluationbadge.css

```diff
@@ -4,8 +4,9 @@
  */
 
 :root {
-	--ck-evaluation-badge-font-size: 7.5px;
-	--ck-evaluation-badge-line-height: 7.5px;
+	--ck-evaluation-badge-font-size: 0.576923em;
+	--ck-evaluation-badge-line-height: 0.576923em;
+	--ck-evaluation-badge-letter-spacing: -0.015em;
```

Comment:
Why not relative to `--ck-font-size-base`?

Reply:
We cannot do it like this:

```css
calc(7.5px / var(--ck-font-size-base))
```

Because in CSS calc() division the right side must be a `<number>` therefore unit based values cannot be used in division like this. (`--ck-font-size-base` is `13px`)